server/app/routers/ai.py

```
@@ -0,0 +1,39 @@
+        tmp_path = tmp.name
+    try:
+        res = analyzer.analyze_image(tmp_path)
+        return JSONResponse(res)
+    finally:
+        try:
+            os.remove(tmp_path)
+        except Exception:
+            pass
+
+@router.post("/analyze-video")
+async def analyze_video(file: UploadFile = File(...), durationSec: int | None = Form(None)):
```

Ruff (0.12.2)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)